src/main/java/com/celements/javascript/JsFileEntry.java

```diff
@@ -72,7 +76,11 @@ public boolean isValid() {
   }
 
   public boolean isModule() {
-    String fileName = UriBuilder.fromUri(getFilepath()).build().getPath();
+    String fileName = getFilepath();
+    Matcher matcher = Pattern.compile(".*/([^/?]+).*").matcher(getFilepath());

```

Comment:
I'm generally wary of writing manual parsers when there's already a library for it. If it's only about the leading `:` we add for marking onDisk paths, why not just use a `replaceFirst("^:", "")` before handing it to the `UriBuilder`? I think it would be better readable and less error prone then using a regex for path/query matching.